.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,6 +30,15 @@
 
 :beginner: If you have a lot to say, be more detailed in this section.
 
+# Testing
+<!--
+    Talk about the testing you have (and have not) done. It's useful
+    for others to know what you've already checked so that they can
```

Comment:
To follow the above idea of test coverage, it includes both automated and manual testing when possible:

```suggestion
    Talk about the testing you have done, and the testing you have not done
    whether you rely on automated tests or you don't know how to test something.
    It's useful for others to know what you've already checked so that they can
```